src/functions/severa/get-flextime-by-user/mockData.json

```diff
@@ -0,0 +1,7 @@
+{
+  "headers": {
+    "Content-Type": "application/json"
+  },
+  "body": "{\"name\": \"Frederic\"}",
+  "pathParameters": { "severaUserId": "83ebc1fe-16a4-b070-975f-e21c4334197b", "endDate": "2021-11-03" }
```

Comment:
i guess Josh would only want 1 mockData since there are 2 in list-allocation and get-flextime  , also, you should remove the value for severaUserId and replace with such as "Insert severa user id here" (Josh commented this with my PR before)